twilio/task_router/__init__.py

```diff
@@ -209,13 +227,15 @@ def __init__(self, account_sid, auth_token, workspace_sid, worker_sid):
                                                          workspace_sid,
                                                          worker_sid)
 
-        self.reservations_url = self.base_url + "/Tasks/**"
         self.activity_url = self.base_url + "/Activities"
+        self.tasks_url = self.base_url + "/Tasks/**"
```

Comment:
Correct, but I'm not sure anyone would directly reference reservations_url since this variable's only purpose would be for generating access policies.

I can revert this if necessary though.